src/js/ideogram.js

```diff
@@ -2129,6 +2129,90 @@ Ideogram.prototype.getTaxids = function() {
   return taxids;
 };
 
+/**
+ * Returns names and lengths of chromosomes fir an organism's best-known genome assembly
+ */
+Ideogram.prototype.getChromosomes = function(organismName) {
+    var output = [];
+    console.log("organismName is: ", organismName);
+    var organism = organismName.split(" ");
+    console.log("organism array is: ", organism);
+    // Querying NCBI database for genome
+    var link = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=assembly&retmode=json&term=";
+    for (var i = 0; i < organism.length; i++) {
+        link += organism[i];
+        link += "%20";
+    }
+    link += "AND%20(%22latest%20refseq%22[filter])%20AND%20%22chromosome%20level%22[filter]";
+    d3.xhr(link, function(data) {
```

Comment:
I originally recommended `d3.xhr`, but it occurs to me `d3.json` would be better for these requests.  That way, D3 will parse the response as JSON without us having to invoke `JSON.parse`.  Please replace `d3.xhr` calls in this method with `d3.xhr`, and remove those calls to `JSON.parse`.